react/Reviews.tsx

```diff
@@ -860,6 +862,7 @@ function Reviews() {
                           more="Show more"
                           less="Show less"
                           anchorClass=""
+                          className={`${handles.showMoreButton}`}
```

Comment:
The `ShowMore` component does not accept a `className` prop. This is currently causing the app build to fail. I believe you should use the `anchorClass` prop instead. Please verify your changes by linking the app in a workspace.

Reply:
Thank's for your review. I will fix this code and try to build the app again. @polishq Thank you!